frontend/javascripts/oxalis/shaders/utils.glsl.ts

```diff
@@ -113,10 +113,12 @@ export function jsColormapJet(x: number): Vector3 {
 
   return [r, g, b];
 }
-export const hslaToCSS = (hsla: Vector4) => {
-  const [h, s, l, a] = hsla;
-  return `hsla(${360 * h}, ${100 * s}%, ${100 * l}%, ${a})`;
+
+export const rgbaToCSS = (rgba: Vector4) => {
```

Comment:
We save color values as RGBA, not HSLA. There is no reason to convert between the color models just for some CSS.